docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -1323,7 +1323,8 @@ application containers can simply mount the Alluxio FileSystem.
 
 #### FUSE daemon
 
-One way to use the POSIX API is to deploy the Alluxio FUSE daemon.
+One way to use the POSIX API is to deploy the Alluxio FUSE daemon, creating pods running Alluxio Fuse processes inside
```

Comment:
inside?

Reply:
I meant the processes run inside the pods. Removed.